doc/snek.adoc

```diff
@@ -520,15 +534,16 @@ within the Tuple is followed by a comma:
 ----
 === Dictionaries
 
-Dictionaries are the fanciest data structure in Snek. Like Lists and
-Tuples, Dictionaries hold multiple values. Unlike those, Dictionaries
-are not indexed by numbers. Instead, Dictionaries are index by
-another Snek value. The only requirement is that the value be
-unchanging, which means Dictionaries can only be indexed by immutable
-values. Lists and Dictionaries are the only mutable data structures in
-Snek, so there are lots of options for Dictionary indices.(((Dictionary)))
+Dictionaries are the fanciest data structure in Snek. Like
+Lists and Tuples, Dictionaries hold multiple values. Unlike
+Lists and Tuples, Dictionaries are not indexed by
+numbers. Instead, Dictionaries are indexed by another Snek
+value. The only requirement is that the index value be immutable,
+so that it can never change. Lists and Dictionaries are the only mutable data
+structures in Snek, so there are lots of options for
```

Comment:
'so there are lots of options' doesn't seem related to 'the only mutable data structures in Snek'. Oh, I think what I was trying to say is that you can use anything other than a list or dictionary as a dictionary index. This is really unclear here.

Reply:
Fixed.